.lighthouserc.js

```diff
@@ -0,0 +1,232 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Configuration for lighthouse-ci.
```

Comment:
is it recommended to use .js for `rc` file? Also, if it's a js file why the name starts with `.`?

Reply:
> is it recommended to use .js for `rc` file? 

It is not recommended, we are free to use either JSON, JS or YAML. I find JS useful because we can add comments to the configuration which are quite useful.

> Also, if it's a js file why the name starts with `.`?

Not sure, I think that other configs have `.` and I don't think it is a problem that it is a JS file.